scripts/build/TestPlatform.Settings.targets

```diff
@@ -49,6 +49,11 @@
   <!-- Package dependency versions -->
   <Import Project="$(MSBuildThisFileDirectory)TestPlatform.Dependencies.props" Condition=" '$(DependencyVersionsImported)' != 'true' "/>
 
+  <!-- Allows build of .NET Framework assemblies on MacOS and Linux https://github.com/dotnet/designs/pull/33#issuecomment-489264196 -->
+  <ItemGroup>
+    <PackageReference Condition=" '$(OS)' != 'Windows_NT' " Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0" PrivateAssets="All" />
```

Comment:
Any reason not to target `1.0.2`? (see https://www.nuget.org/packages/Microsoft.NETFramework.ReferenceAssemblies)

Reply:
Nope, I just did not check what is the latest, because they kept all the other packages we used before stick to 1.0.0.

Reply:
```suggestion
    <PackageReference Condition=" '$(OS)' != 'Windows_NT' " Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.2" PrivateAssets="All" />
```